lib/jake/array.ex

```diff
@@ -1,5 +1,5 @@
 defmodule Jake.Array do
-  def gen(spec) do
+  def gen(spec, context) do
```

Comment:
in all the gen cases, is spec same as context.child? if so can we refactor it to `gen(%Context{child: spec} = context)`. Do we need to pass two args?

Reply:
yes, I was also thinking the same. I thought may be it will break pattern matching in gen(). I'll look into it.